phpBB/phpbb/auth/provider/oauth/oauth.php

```diff
@@ -172,6 +172,7 @@ public function login($username, $password)
 		$service_name = $this->get_service_name($provider);
 
 		if ($provider === '' || !array_key_exists($service_name, $this->service_providers))
+
```

Comment:
hm?

Reply:
outdated. Was left over from resolving conflicts.
Was found by travis' code sniff and fixed in https://github.com/phpbb/phpbb/pull/5584/commits/294528cacec5236e71a84c2019572d7ba9321244